pkg/oc/bootstrap/clusterup/components/registry/registry_install.go

```diff
@@ -42,7 +41,15 @@ func (r *RegistryComponentOptions) Name() string {
 }
 
 func (r *RegistryComponentOptions) Install(dockerClient dockerhelper.Interface, logdir string) error {
-	kubeClient, err := kubernetes.NewForConfig(r.ClusterAdminKubeConfig)
+	clusterAdminKubeConfigBytes, err := ioutil.ReadFile(path.Join(r.MasterConfigDir, "admin.kubeconfig"))
```

Comment:
Can we make something like `ComponentContext` and have all clients there? Guess we will have to repeat this in all installs.

Reply:
I mean something like:

```go
client, err := r.GetAdminKubeClient()
```